README.md

```diff
@@ -50,6 +50,7 @@ Contributions, issues, and feature requests are welcome!
 Give a ⭐️ if you like this project!
 
 ## Acknowledgments
+  attribution note-Thanks to Abdul and Basit for allowing to use their code.
 
 
 ## 📝 License
```

Comment:
The license link is not working because there is no LICENSE file in you root folder, kindly create a License file and you may want to use this [MIT format](https://gist.github.com/adamveld12/b64e74d14d7d9043b85c) and customize it. The MIT license gives users express permission to reuse code for any purpose, sometimes even if code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs. 👍